File: practice/algorithms/strings/camelcase/CamelCase.js
'use strict';

const processData = input => {
    const line = input;
    const words = line.split(/[A-Z]/g);
    console.log(words.length);
};

process.stdin.resume();
process.stdin.setEncoding("ascii");

let _input = "";
process.stdin.on("data", input => _input += input);
process.stdin.on("end", () => processData(_input));

File: practice/algorithms/strings/camelcase/CamelCase.java
import java.util.*;

public class CamelCase {

    public static void main(String[] args) {
        Scanner stdin = new Scanner(System.in);
        String text = stdin.nextLine();
        String[] words = text.split("[A-Z]");
        System.out.println(words.length);
        stdin.close();
    }
}